lms/djangoapps/shoppingcart/models.py

```diff
@@ -1545,12 +1636,14 @@ def contained_in_order(cls, order, course_id):
         ]
 
     @classmethod
-    def get_total_amount_of_purchased_item(cls, course_key):
+    def get_total_amount_of_purchased_item(cls, course_key, status='purchased'):
```

Comment:
I have added the methods related to the invoices in the `InvoiceHistoryTest` class and the order related method tests in the `PaidCourseRegistrationTests` class.